### DISCUSSION TITLE
List of Nucleotides for each Node

### DISCUSSION THREAD

**Discussion by AlirezaPa** [tskit-dev/msprime#1890] [Q&A] [answered]
Hi msprime community,

I have a simple question. Is there any way to see the nucleotides (ACCGGTC...) for the following command and tree. I expect to see 10 nucleotides for each node. No mutation rate and no recombination rate.

ts = msprime.sim_ancestry(2, sequence_length=10)

![image](https://user-images.githubusercontent.com/93855073/140636311-0b6599f8-e2c9-4f5f-8284-9254c443f0ea.png)

Thank you,
Alireza

**Comment by jeromekelleher** [comment, marked as answer]
Hi @AlirezaPa, welcome :wave:

The [sim_ancestry](https://tskit.dev/msprime/docs/stable/ancestry.html) function only simulates the ancestral trees for a sample (like shown above). To get sequence data we have to simulate mutations on top of this ancestry, using the [sim_mutations](https://tskit.dev/msprime/docs/stable/mutations.html) function. Hope this helps!
